### ISSUE TITLE
p2p sats book [kind 38383] Positive Interoperability Tracker

### ISSUE THREAD

**Issue by alltheseas** [nostrability/nostrability#104]
https://github.com/nostr-protocol/nips/pull/1331

**app**|**see kind-38382**|**post kind-38383**|**comment**
-----|-----|-----|-----
mostro|✅|✅|-
robosats|✅|✅|-

**Comment by alltheseas** [comment]
@koalasat can you confirm which apps follow p2p spec?

**Comment by KoalaSat** [comment]
> @KoalaSat can you confirm which apps follow p2p spec?

The Robosats part if correct.

As far I know mostro uses https://github.com/lnp2pBot/bot on the back, so we can consider that both makes use of it. But for sure @grunch can confirm this.

**Comment by grunch** [comment]
Yes [@lnp2pbot](https://github.com/lnp2pBot/bot) also is publishing nip-69 events

**Comment by alltheseas** [comment]
hodlhodl investigating being listed on p2p.band:

https://njump.me/nevent1qqsws0p6355e2s46t63wjjftrr7zmwscyhnf4r9ee79fj302qsyz2uqml0pqz

**Comment by alltheseas** [comment]
https://nostr.eu/nevent1qqsf0fsmws0849guw8uk8gs2cz97raps30gg7xz4kexufmv9rcgvrfchknzkg

> p2p.band is down and probably is because is receiving flood of orders from @hodlhodl , I'm running it locally and currently we have over 10k NIP-69 pending nostr orders from hodlhodl, this is starting to be a problem to relays, which may start blocking orders from hodlhodl pubkey, please holdhodl team give it a look ASAP
